uw-frame-components/portal/main/controllers.js

```diff
@@ -109,7 +109,7 @@ define(['angular', 'require'], function(angular, require) {
 
       if (vm.user.firstName || vm.user.displayName) {
         vm.username = vm.user.firstName ?
-          vm.user.firstName : vm.user.displayName;
+          vm.user.firstName.toLowerCase() : vm.user.displayName.toLowerCase();
```

Comment:
It's necessary to change the case of the username in the menu. It has to be programmatically changed to lowercase because the CSS `text-transform: capitalize` has no effect on capital letters. I figured using this with CSS is less overkill than making a custom filter.